mockingcase.test.js

```diff
@@ -200,4 +201,184 @@ describe("mockingcase", () => {
       expect(consoleOutput).toEqual(expectedOutput);
     });
   });
+
+  describe(`option.upper | set as String`, () => {
+    const lowerCaseInput = "hello world";
+    const capitalizeInput = "Hello World";
+    const upperCaseInput = "HELLO WORLD";
+
+    test("it should make match substring upper case", () => {
```

Comment:
`it` is not necessary here as tests can be written also with `it("...`. To not make changes everywhere will stick with `test("...` but in my opinion `it` is redundant here.
```suggestion
    test("Should make match substring upper case", () => {
```